db/migrate/20220224202927_add_users_ref_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddUsersRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
+  end
+end
```

Comment:
- [ ] While creating the likes, we need the user or `author_id and post_id` to be included in the comment table and a corresponding index

Reply:
@Emmyn5600  can yo please clarify on this